ExtraDry/ExtraDry.Blazor/Components/Special/Theme.razor.cs

```diff
@@ -31,14 +31,38 @@ public partial class Theme : ComponentBase {
     [Parameter]
     public Type? ErrorComponent { get; set; }
 
+    /// <inheritdoc cref="ValueLoader{ValueModel}.ErrorIndicator" />
+    /// <see cref="ValueLoader{ValueModel}"/>
+    [Parameter]
+    public RenderFragment<IndicatorContext>? ErrorIndicator { get; set; }
+    /// <inheritdoc cref="ValueLoader{ValueModel}.TimeoutIndicator" />
+    /// <see cref="ValueLoader{ValueModel}"/>
+    [Parameter]
+    public RenderFragment<IndicatorContext>? TimeoutIndicator { get; set; }
```

Comment:
Could there ever be another TimeoutIndicator?  E.g. a GraphView?  Maybe that takes something that isn't a IndicatorContext?  If the component becomes named "Foo", then maybe this should be the "FooTimeoutContent"?